templates/default/variables.twig

```diff
@@ -1,38 +1,5 @@
-{% set translations = {
-    'authentication.start_button': 'authentication.start_button' | trans,
-    'registration.start_button': 'registration.start_button' | trans,
-    'registration.sub_title':  'registration.sub_title' | trans,
-    'status.registration_initial':  'status.registration_initial' | trans,
```

Comment:
The no longer used translations (for the old react app) have been removed from the template.